test/powershell/Modules/Microsoft.PowerShell.Utility/Get-Uptime.Tests.ps1

```diff
@@ -19,29 +19,20 @@ Describe "Get-Uptime" -Tags "CI" {
     }
     It "Get-Uptime return timespan (default -Timespan)" {
         $upt = Get-Uptime
-        $upt | Should BeOfType "Timespan"
+        $upt | Should -BeOfType "Timespan"
     }
     It "Get-Uptime -Since return DateTime" {
         $upt = Get-Uptime -Since
-        $upt | Should BeOfType "DateTime"
+        $upt | Should -BeOfType "DateTime"
     }
     It "Get-Uptime throw if IsHighResolution == false" {
-        try
         {
             # Enable the test hook
             [system.management.automation.internal.internaltesthooks]::SetTestHook('StopwatchIsNotHighResolution', $true)
 
             Get-Uptime
-            throw "No Exception!"
-        }
-        catch
-        {
-            $_.FullyQualifiedErrorId | Should be "GetUptimePlatformIsNotSupported,Microsoft.PowerShell.Commands.GetUptimeCommand"
-        }
-        finally
-        {
-            # Disable the test hook
-            [system.management.automation.internal.internaltesthooks]::SetTestHook('StopwatchIsHighResolutionIsFalse', $false)
-        }
+        } | Should -Throw -ErrorId "GetUptimePlatformIsNotSupported,Microsoft.PowerShell.Commands.GetUptimeCommand"
+        # Disable the test hook
+        [system.management.automation.internal.internaltesthooks]::SetTestHook('StopwatchIsHighResolutionIsFalse', $false)
```

Comment:
I think you still want the disabling of the test hook in a `finally` block to ensure it runs

Reply:
Just wrap the test execution line in try{} and disabling the test hook in finally{}